proposals/rkt.adoc

```diff
@@ -0,0 +1,81 @@
+## rkt
+
+*Name of project*: rkt
+
+*Description*:
+
+rkt is an application container engine developed for modern production cloud-native environments. It features a pod-native approach, a pluggable execution architecture, and a well-defined surface area that makes it ideal for integration with other systems.
+
+The core execution unit of rkt is the _pod_, a collection of one or more applications executing in a shared context. (rkt's pods are synoymous with the concept in the Kubernetes orchestration system). rkt allows users to apply different configurations (like isolation parameters) at both pod-level and at the more granular per-application level. rkt's architecture means that each pod executes directly in the classic Unix process model (i.e. there is no central daemon), in a self-contained, isolated environment.
```

Comment:
Hm, this paragraph does lead with an explanation of what pods are. I've added a link to the Kubernetes pod definition too. Does that help? (I honestly really dislike the pod/container conflation so don't want to repeat it here)